backend/functions/src/index.ts

```diff
@@ -2,15 +2,21 @@ import * as functions from 'firebase-functions'
 import admin from 'firebase-admin';
 import express from 'express';
 import cors from 'cors';
+
+import students from './routes/students/index';
+
 //import bodyParser from 'body-parser';
 const serviceAccount: string = require('../service-account.json')
 
 // initialize firebase inorder to access its services
-admin.initializeApp({credential: admin.credential.cert(serviceAccount)});
+const firebaseApp = admin.initializeApp({credential: admin.credential.cert(serviceAccount)});
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use("/students", students)
 
-//initialize express server
-const app = express()
-app.use(express.json())
-app.use(cors())
+exports.api = functions.https.onRequest(app)
 
-exports.api = functions.https.onRequest(app)
\ No newline at end of file
+export {firebaseApp};
```

Comment:
There should be a `db` variable in this file now if you pull from `main`. To access the database, using the `db` instance is preferred to accessing the entire app.